rllib/algorithms/registry.py

```diff
@@ -229,6 +229,7 @@ def _import_leela_chess_zero():
     "DreamerV3": _import_dreamerv3,
     "DT": _import_dt,
     "IMPALA": _import_impala,
+    "Impala": _import_impala,
```

Comment:
wait, where is this coming from? It will mess up with our telemetry.

Reply:
Let me explain: We added a new test case in this PS, which is IMPALA (separate policy and vf) on cartpole. The new `tuned_example` file is a python file (I'm trying to create as few new yamls as possible nowadays). Hence, in there I'm using the `ImpalaConfig()` class/object. It seems to not work well with `tune.run_experiment` for whatever reason.

I didn't think about telemetry. Let me see, whether there is a better way that would not break things ...

Reply:
Ah, ok, this is the culprit here (in `rllib/train.py`).

```
        experiments = {
            f"default_{uuid.uuid4().hex}": {
                "run": algo_config.__class__.__name__.replace("Config", ""),
                "env": config.get("env"),
                "config": config,
                "stop": stop,
            }
        }
```

Ok, let me provide a better fix.

Reply:
yeah We should import the name directly from the registry if possible or avoid run.run_experiments?

Reply:
All algo config objects know what their corresponding algo class is, so this is solved now much more elegantly.